.travis.yml

```diff
@@ -41,7 +41,7 @@ script:
     - pip install -q git+https://github.com/OCA/openupgradelib.git@master
     - createdb $DB
     - wget -q -O- https://github.com/OCA/OpenUpgrade/releases/download/databases/8.0.psql | pg_restore -d $DB --no-owner
-    - MODULES=base,$(sed -n '/^+========/,$p'  openerp/openupgrade/doc/source/modules80-90.rst | grep Done | sed -n '/^|/ s/^|\([0-9a-z_]*\) *|.*$/\1/g p' | paste -d, -s)
+    - MODULES=base,$(sed -n '/^+========/,$p'  openerp/openupgrade/doc/source/modules80-90.rst | grep "Done\|Partial" | sed -n '/^|/ s/^|\([0-9a-z_]*\) *|.*$/\1/g p' | paste -d, -s)
```

Comment:
It seems to be working fine though https://travis-ci.org/OCA/OpenUpgrade/builds/138224440#L307